flow/network.h

```diff
@@ -201,7 +201,7 @@ namespace std
 		{
 			int result = 0;
 			if (na.ip.isV6()) {
-				result = crc32c_append( 0xfdbeefdb, (uint8_t*)na.ip.toV6().data(), 16 );
+				result = ((uint16_t*) na.ip.toV6().data())[7];
```

Comment:
Considering `size_t` is `ulong` i.e. 64bit, maybe we can return last 64bit of address? Ideally, that should have better spread.

Reply:
good suggestion